package.json

```
@@ -7,7 +7,7 @@
     "url": "https://github.com/sourcegraph/sourcegraph"
   },
   "engines": {
-    "node": ">=8.0.0 <14.0.0",
+    "node": ">=8.0.0 <=14.1.0",
```

Just realised that we probably rather just exclude 15. If that is correct, please can you apply this suggestion?

```suggestion
    "node": ">=8.0.0 <15.0.0",
```